packages/website/posts/2022-03-29-50M-uploads.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: 'NFT.Storage crosses 50M uploads'
+description: NFT.Storage has grown to 50M uploads. Here's to the next 500M!
+author: David Choi
+thumbnail: https://user-images.githubusercontent.com/89210666/160478599-0de7cfdd-da01-4df5-a24d-12dcc01db259.png
+date: Mar 29, 2022
+tags:
+  - milestone
+---
+
+We're excited to announce that over the weekend, NFT.Storage crossed 50 million user uploads to our service! It's a big milestone that shows both the overall growth of the NFT space, as well as increased awareness within the space of the importance of handling off-chain NFT data the right way. The NFT ecosystem is increasingly seeing the importance of making references off-chain data [content addressed](https://docs.ipfs.io/concepts/content-addressing/) (using IPFS CIDs so no one can argue what the NFT is for), and ensuring this data is always accessible on IPFS via these content addresses (with NFT.Storage as one perpetual storage option with long-term provable storage via Filecoin, but also utilizing IPFS's flexibility to store wherever else [makes them feel comfortable](https://nft.storage/blog/post/2021-12-14-storage-layer-maximalism/)).
+
+# Young but precocious
+
+NFT.Storage was launched as a free storage service at the [end of April, 2021 by Protocol Labs](https://filecoin.io/blog/posts/introducing-nft.storage-free-decentralized-storage-for-nfts/) to help NFT creators and minting services follow best-practices with handling their off-chain NFT data, a piece that is usually as critical as the on-chain piece but often overlooked. We wanted for it to be easy for users to trustlessly get the IPFS CID of their off-chain content and make it available on the IPFS network perpetually, with copies of the data stored on the Filecoin network (meaning users could always check the Filecoin blockchain for validated proofs that their data is physicaly stored without needing to trust us).
+
+We're excited to announce that, in less than a year, we've crossed 50 million user uploads to our service! This roughly translates to 50 million NFTs, though it's likely to be many more (since one directory uploaded often corresponds to many NFTs, like for 10K PFP drops). Since launch, this translates to:
```

Comment:
"This roughly translates to 50 million NFTs" -> "This is at minimum 50 million NFTs"

Reply:
strictly this isn't necessarily true (can have multiple uploads for an NFT if you store metadata and assets separately) but i'll massage the language